cli/fossilize_replay.cpp

```diff
@@ -2163,17 +2153,8 @@ struct ThreadedReplayer : StateCreatorInterface
 		if (opts.pipeline_stats)
 			info->flags |= VK_PIPELINE_CREATE_CAPTURE_STATISTICS_BIT_KHR;
 
-		auto *library_info = find_pnext<VkPipelineLibraryCreateInfoKHR>(
-				VK_STRUCTURE_TYPE_PIPELINE_LIBRARY_CREATE_INFO_KHR, create_info->pNext);
-		bool consumes_libraries = library_info && library_info->libraryCount != 0;
 		bool generates_library = (create_info->flags & VK_PIPELINE_CREATE_LIBRARY_BIT_KHR) != 0;
 
-		if (generates_library && consumes_libraries)
```

Comment:
I only see code that attempts to do this for RT.

Reply:
`work_item_get_parent_depth` should handle all pipeline types. Are there other code paths that miss an implementation?

Reply:
You're not setting hash_to_index[hash] for those at least. In the code you're looking at the create info to determine the depth, but how can you know the create info when the pipeline has not even been parsed yet?

Reply:
I added the missing hash_to_index (hash_to_info now) handling. Parsing pipeline libraries should be fixed as well.